src/computational_graph/graphvector.jl

```diff
@@ -0,0 +1,86 @@
+"""
+    struct GraphVector{F,W} <: AbstractVector{Graph{F,W}}
+
+A vector of graphs. All graphs must have the same number of external legs, but can have different external labels.
+
+Support two constructors:
+    GraphVector(graphs::AbstractVector) # construct from a vector of Graph,  use the default factor and weight types
+    GraphVector{F,W}(graphs::AbstractVector) # construct from a vector of Graph,  use the specified factor and weight types `F` and `W`
+"""
+struct GraphVector{F,W} <: AbstractVector{Graph{F,W}}
```

Comment:
Why do we need this new type? What's the difference between this and simply
const GraphVector{F,W} = Vector{Graph{F,W}}?